src/screens/points/content/PointsContent.tsx

```diff
@@ -747,16 +747,16 @@ export function PointsContent() {
           </Inset>
           {!shouldDisplayError ? (
             <Inset horizontal="20px" top="24px">
-              <Stack space="28px">
-                <Stack space="20px">
+              <Box gap={28}>
+                <Box gap={20}>
                   <Text color="label" size="20pt" style={{ marginLeft: 4 }} weight="heavy">
                     {i18n.t(i18n.l.points.points.my_points)}
                   </Text>
                   <Box flexDirection="row" alignItems="center" paddingLeft="4px">
                     {canDisplayTotalPoints ? <RainbowText totalPointsString={totalPointsString} /> : <Skeleton height={31} width={200} />}
                   </Box>
-                </Stack>
-                <Box gap={24}>
+                </Box>
+                <Box width="full" gap={24}>
```

Comment:
giving this box `width="full"` fixed the issue. Converted all the stacks to Boxes because Stack's cannot take width prop